ethereum/p2p/src/test/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryTimestampsTest.java

```diff
@@ -41,25 +40,25 @@ public void lastSeenAndFirstDiscoveredTimestampsUpdatedOnMessage() {
     final Packet pong = helper.createPongPacket(agent, Hash.hash(agentPing.getHash()));
     helper.sendMessageBetweenAgents(testAgent, agent, pong);
 
-    final AtomicLong lastSeen = new AtomicLong();
-    final AtomicLong firstDiscovered = new AtomicLong();
+    long lastSeen;
```

Comment:
Can't see why an AtomicLong was ever needed, but it's normally a very deliberate choice. Is there some concurrency happening in this test?

Reply:
@jframe  I checked the git history. It used to have code similar to:
```
await()
        .atMost(1, TimeUnit.SECONDS)
        .untilAsserted(
            () -> {
              assertThat(controller.getPeers()).hasSize(1);
```
which since has been removed. Now there is no need to have AtomicLong anymore (and errorprone reported it as `UnnecessaryAsync` warning).